apps/ui/src/components/Modal/SelectValidation.vue

```diff
@@ -145,6 +145,7 @@ const definition = computed(() => {
 
   if (selectedValidation.value.key === 'basic') {
     updated.properties.minScore.examples = ['e.g. 1.23'];
+    delete updated.properties.strategies;
```

Comment:
This is to remove the `strategies` object from the definition, so the UiForm does not show the related inputs for those fields, since this form has a special component to handle sub-strategies